### PR TITLE
en: add single page to describe enterprise edition deployment

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)

Add a single document to describe how to deploy enterprise edition TiDB cluster.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/638
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: en/deploy-tidb-enterprise-edition.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Deploy TiDB Enterprise Edition in Kubernetes
+summary: Learn how to deploy TiDB Enterprise Edition in Kubernetes.
+---
+
+# Deploy TiDB Enterprise Edition in Kubernetes
+
+This document describes how to deploy TiDB Cluster Enterprise Edition and related tools in Kubernetes. TiDB Enterprise Edition has the following features:
```
Comment (ran-huang) [inline en/deploy-tidb-enterprise-edition.md:8]:
```suggestion
This document describes how to deploy TiDB Enterprise Edition and related tools in Kubernetes. TiDB Enterprise Edition has the following features:
```

File: en/deploy-tidb-enterprise-edition.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* Enterprise-level service support
+* Fully enhanced security features
+
+## Prerequisites
+
+* TiDB Operator is [deployed](deploy-tidb-operator.md).
+  
+## Deploy method
+
+Currently, the difference between the deployment of TiDB Operator Enterprise Edition and Community Edition is mainly reflected in the image naming. Compared with Community Edition, the image of Enterprise Edition has an extra `-enterprise` suffix.
```
Comment (ran-huang) [inline en/deploy-tidb-enterprise-edition.md:20]:
```suggestion
Currently, the difference between the deployment of TiDB Operator Enterprise Edition and Community Edition is mainly reflected in the image naming. Compared with that of Community Edition, the image of Enterprise Edition has an extra `-enterprise` suffix.
```

File: en/deploy-tidb-enterprise-edition.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    baseImage: pingcap/tiflash-enterprise
+  ...
+  pump:
+    baseImage: pingcap/tidb-binlog-enterprise
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc-enterprise
+```
+
+If you are deploying a new cluster, refer to [Configure a TiDB Cluster in Kubernetes](configure-a-tidb-cluster.md) to configure `tidb-cluster.yaml` and the enterprise edition image as described above, and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace }` command to deploy TiDB Enterprise Edition cluster and related tools.
```
Comment (ran-huang) [inline en/deploy-tidb-enterprise-edition.md:45]:
```suggestion
If you are deploying a new cluster, refer to [Configure a TiDB Cluster in Kubernetes](configure-a-tidb-cluster.md) to configure `tidb-cluster.yaml` and the enterprise edition image as described above, and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace}` command to deploy the TiDB Enterprise Edition cluster and related tools.
```

File: en/deploy-tidb-enterprise-edition.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+If you need to switch an existing cluster to Enterprise Edition:
+
+**Method #1**: Add the "-enterprise" suffix to the `baseImage` item of the configuration file of the existing cluster in the above format and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace}` command to update the cluster configuration.
+**Method #2**: Add the suffix "-enterprise" to each component `baseImage` in the above format and run the `kubectl edit tc ${name} -n ${namespace}` command to update the cluster configuration.
```
Comment (ran-huang) [inline en/deploy-tidb-enterprise-edition.md:50]:
```suggestion
- **Method #2**: Run the `kubectl edit tc ${name} -n ${namespace}` command to add the suffix "-enterprise" to each component `baseImage` in the above format, and then update the cluster configuration.
```
@lichunzhu PTAL whether the suggestion is correct.

File: en/deploy-tidb-enterprise-edition.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  pump:
+    baseImage: pingcap/tidb-binlog-enterprise
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc-enterprise
+```
+
+If you are deploying a new cluster, refer to [Configure a TiDB Cluster in Kubernetes](configure-a-tidb-cluster.md) to configure `tidb-cluster.yaml` and the enterprise edition image as described above, and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace }` command to deploy TiDB Enterprise Edition cluster and related tools.
+
+If you need to switch an existing cluster to Enterprise Edition:
```
Comment (ran-huang) [inline en/deploy-tidb-enterprise-edition.md:47]:
```suggestion
If you need to switch an existing Community Edition cluster to Enterprise Edition:
```

File: en/deploy-tidb-enterprise-edition.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+    baseImage: pingcap/tiflash
+  ...
+  pump:
+    baseImage: pingcap/tidb-binlog
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc
+```
+
+If you need to switch an existing cluster back to Community Edition:
```
Comment (ran-huang) [inline en/deploy-tidb-enterprise-edition.md:79]:
```suggestion
If you need to switch an existing Enterprise Edition cluster back to Community Edition:
```

File: en/deploy-tidb-enterprise-edition.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc-enterprise
+```
+
+If you are deploying a new cluster, refer to [Configure a TiDB Cluster in Kubernetes](configure-a-tidb-cluster.md) to configure `tidb-cluster.yaml` and the enterprise edition image as described above, and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace }` command to deploy TiDB Enterprise Edition cluster and related tools.
+
+If you need to switch an existing cluster to Enterprise Edition:
+
+**Method #1**: Add the "-enterprise" suffix to the `baseImage` item of the configuration file of the existing cluster in the above format and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace}` command to update the cluster configuration.
```
Comment (ran-huang) [inline en/deploy-tidb-enterprise-edition.md:49]:
```suggestion
- **Method #1**: Add the "-enterprise" suffix to the `baseImage` item of the configuration file of the existing cluster in the above format and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace}` command to update the cluster configuration.
```

File: en/deploy-tidb-enterprise-edition.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+    baseImage: pingcap/tidb-binlog
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc
+```
+
+If you need to switch an existing cluster back to Community Edition:
+
+**Method #1**: Remove the "-enterprise" suffix from the `baseImage` item in the configuration file of the existing cluster in the above format and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace}` command to update the cluster configuration.
+**Method #2**: Remove the suffix "-enterprise" to each component `baseImage` in the above format and run the `kubectl edit tc ${name} -n ${namespace}` command to update the cluster configuration.
```
Comment (ran-huang) [inline en/deploy-tidb-enterprise-edition.md:82]:
```suggestion
- **Method #1**: Remove the "-enterprise" suffix from the `baseImage` item in the configuration file of the existing cluster in the above format, and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace}` command to update the cluster configuration.
- **Method #2**: Run the `kubectl edit tc ${name} -n ${namespace}` command to remove the suffix "-enterprise" from each component's `baseImage` in the above format, and then update the cluster configuration.
```

**Comment by lilin90** [comment]
@lonng FYI.

File: en/deploy-tidb-enterprise-edition.md (hunk 3 above)
Comment (lilin90) [reply en/deploy-tidb-enterprise-edition.md:45]: Suggest using a list for the following section: deploy a new cluster, switch from Community Edition.

File: en/deploy-tidb-enterprise-edition.md (hunk 4 above)
Comment (lilin90) [reply en/deploy-tidb-enterprise-edition.md:50]: @ran-huang I think your version is right.

File: en/deploy-tidb-enterprise-edition.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+If you are deploying a new cluster, refer to [Configure a TiDB Cluster in Kubernetes](configure-a-tidb-cluster.md) to configure `tidb-cluster.yaml` and the enterprise edition image as described above, and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace }` command to deploy TiDB Enterprise Edition cluster and related tools.
+
+If you need to switch an existing cluster to Enterprise Edition:
+
+**Method #1**: Add the "-enterprise" suffix to the `baseImage` item of the configuration file of the existing cluster in the above format and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace}` command to update the cluster configuration.
+**Method #2**: Add the suffix "-enterprise" to each component `baseImage` in the above format and run the `kubectl edit tc ${name} -n ${namespace}` command to update the cluster configuration.
+
+TiDB Operator will automatically update the cluster image to the enterprise image through a rolling upgrade.
+
+## Switch back to Community Edition
```
Comment (lilin90) [inline en/deploy-tidb-enterprise-edition.md:54]:
```suggestion
## Switch back to the Community Edition
```

**Pull request merged**